arceos-vmm/Cargo.toml

```diff
@@ -25,12 +25,13 @@ axstd = { workspace = true, features = [
 # System dependent modules provided by ArceOS-Hypervisor.
 axvm = { workspace = true }
 axvcpu = { workspace = true }
+axaddrspace = { workspace = true }
 
 # System independent crates provided by ArceOS, these crates could be imported by remote url. 
 axerrno = "0.1.0"
-memory_addr = "0.2.0"
+memory_addr = "0.3"
 page_table_entry = { version = "0.3.3", features = ["arm-el2"] }
```

Comment:
Fix `page_table_entry` also please.

Reply:
ok

Reply:
Also please check Cargo.lock to see if there are any crates with multiple versions referenced.

Reply:
> Also please check Cargo.lock to see if there are any crates with multiple versions referenced.

updated